.travis.yml

```diff
@@ -4,7 +4,6 @@ sudo: false
 php:
   - 7.1
   - 7.2
-  - nightly
```

Comment:
Is the removal of nightly here intentional?

It still remains in the `allow_failures` section.

Reply:
It is actual not possible to install the project on PHP 7.2 and Nightly when the CS-Fixer is included. On lines 36-39 the project is installed without php-version-check in composer.

Removing nightly in the matrix saves 2 jobs.